pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -3544,6 +3544,31 @@ The paths themselves are given in the direction of the first geometry.`,
 			Volatility: tree.VolatilityImmutable,
 		},
 	),
+	"st_rotate": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"g", types.Geometry},
+				{"rotRadians", types.Float},
```

Comment:
nit: snake case -- `rot_radians`

Reply:
Just curious, is snake case the standard style at Cockroach?

Reply:
for the builtin arguments name it (mostly) seems to be

Reply:
fixed in recent commit.